core/domain/exp_services.py

```diff
@@ -1514,8 +1576,13 @@ def regenerate_exploration_summary_with_new_contributor(
     """
     exploration = exp_fetchers.get_exploration_by_id(
         exploration_id, strict=False)
+    exp_rights = exp_models.ExplorationRightsModel.get(
```

Comment:
This should be named exp_rights_model, right?

Shouldn't we be converting this to a rights domain object before doing anything with it?

Reply:
Updated.